package.json

```diff
@@ -3,11 +3,14 @@
   "scripts": {
     "build": "buidler compile",
     "test": "buidler test",
+    "gas-reporter": "GAS_REPORTER=1 buidler test --network gasReporter",
+    "testnet": "buidler node",
```

Comment:
@fedgiac - I am under the impression we are no longer using buidler... Is there an alternative for this or do we still need it here?